source/documentation/fuseki2/fuseki-configuration.md

```diff
@@ -124,9 +124,71 @@ An in-memory dataset, with data in the default graph taken from a local file.
         ## tdb:unionDefaultGraph true ;
          .
 
+#### TDB2
+
+    <#dataset> rdf:type      tdb:DatasetTDB2 ;
+        tdb:location "/some/path" ;
+        # this can also have unionDefaultGraph, etc.
+         .
+
 #### Inference
 
-> @@
+An inference reasoner can be layered on top of a dataset as defined above. The type of reasoner must be selected carefully and should not include more reasoning than is required by the application, as extensive reasoning can be detrimental to performance.
+
+You have to build up layers of dataset, inference model, and graph.
+
+    <#dataset> rdf:type ja:RDFDataset;
+         ja:defaultGraph <#inferenceModel>
+         .
+         
+    <#inferenceModel> rdf:type      ja:InfModel;
+         ja:reasoner [ ja:reasonerURL <http://example/someReasonerURLHere> ];
```

Comment:
Maybe make a reference somewhere in the docs that this one here, is one of the values from the reasoners listed in the next section?